starter_project_mobile_2022/lib/blog_detail_murad/presentation/screens/blog_details_page.dart

```diff
@@ -0,0 +1,204 @@
+import 'dart:ui';
+import 'package:flutter/material.dart';
+import 'package:starter_project_mobile_2022/blog_detail_murad/costants.dart/constants.dart';
+import 'package:starter_project_mobile_2022/blog_detail_murad/presentation/widgets/article_detail.dart';
+import 'package:starter_project_mobile_2022/blog_detail_murad/presentation/widgets/comment_diaplay.dart';
+import 'package:starter_project_mobile_2022/blog_detail_murad/presentation/widgets/starBar.dart';
+
+class PostDetailpage extends StatelessWidget {
+  static const routeName = 'postdetail';
+  @override
+  Widget build(BuildContext context) {
+    final double height = MediaQuery.of(context).size.height;
+    final double width = MediaQuery.of(context).size.width;
+    return Scaffold(
+      backgroundColor: Colors.white,
+      appBar: AppBar(
+        title: const Center(
+            child: Text(
+          'Article Detail',
+          style: TextStyle(
+            color: Color.fromARGB(255, 0, 0, 0),
+            fontSize: 25,
+            fontWeight: FontWeight.w100,
+            height: 1.8,
+          ),
+        )),
+        backgroundColor: Colors.transparent,
+        elevation: 0,
+        iconTheme: const IconThemeData(
+          color: Colors.black,
+        ),
+      ),
+      body: SafeArea(
+        minimum: const EdgeInsets.symmetric(vertical: 12),
+        child: ListView(
+          children: [
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  const Expanded(flex: 1, child: Icon(Icons.person)),
+                  Expanded(
+                    flex: 10,
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: const [
+                        Text('Adrian Smith',
+                            style: TextStyle(fontWeight: FontWeight.w800)),
+                        Text('Dec 12 1999')
+                      ],
+                    ),
+                  ),
+                  const Expanded(flex: 1, child: Icon(Icons.more_vert)),
+                ],
+              ),
+            ),
+
+            const Padding(
+              padding: EdgeInsets.all(12.0),
+              child: Text(
+                "Title goes here",
+                style: TextStyle(
+                  color: Colors.black,
+                  fontSize: 32,
```

Comment:
Again, use the following instead of hard-coded size variables.
```
double width = MediaQuery.of(context).size.width
...
padding: EdgeInsets.all(width * 0.02),
...
fontSize: width * 0.001
```